scenes/game/local_game.gd

```diff
@@ -3199,18 +3200,40 @@ func do_remaining_effects(performing_player : Player, next_state):
 	while active_strike.remaining_effect_list.size() > 0:
 		var remaining_effect_count = active_strike.remaining_effect_list.size()
 		if remaining_effect_count > 1:
-			# Send choice to player
-			change_game_state(Enums.GameState.GameState_PlayerDecision)
-			decision_info.type = Enums.DecisionType.DecisionType_ChooseSimultaneousEffect
-			decision_info.player = performing_player.my_id
-			decision_info.choice = active_strike.remaining_effect_list
-			events += [create_event(Enums.EventType.EventType_Strike_EffectChoice, performing_player.my_id, 0, "EffectOrder")]
-			break
+			# Check to see if any of these effects actually have their condition met (or have a negative condition).
+			# If more than 1, send only those choices to the player.
+			# If only 1 does, remove it from the list and do it immediately.
+			# If none do, this is over, clear out the list.
```

Comment:
Even if we decide we want to show the player effects with unmet conditions, maybe it could be nice to repurpose this to indicate that somehow (e.g. changing the button color?) Clearing the list when nothing is met is probably safe either way though yeah

Reply:
Agreed. This will still be useful later.